src/main/resources/lib/ssb/parts/publicationArchive.ts

```diff
@@ -133,7 +133,7 @@ function prepareStatisticRelease(
 
     return {
       title: language === 'en' ? release.statisticNameEn : release.statisticName,
-      period: period.charAt(0).toUpperCase() + period.slice(1),
+      period: period.charAt(0) + period.slice(1),
```

Comment:
It's enough to define `period` here. The rest of the code removes the first letter of the string and re-adds it, which is not necessary.